lbr_examples/rviz_config/ext_torque.rviz

```diff
@@ -0,0 +1,213 @@
+Panels:
```

Comment:
this rviz config adds redundancy to the `lbr_bringup` package, should be removed